qemu/tests/qemu_io_blkdebug.py

```diff
@@ -92,8 +92,8 @@ def run(test, params, env):
         else:
             session.close()
             try:
-                image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
-                output = ""
+                cmd_result = image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
+                output = cmd_result.stdout + cmd_result.stderr
```

Comment:
@PandaWei maybe you misunderstand patch avocado-framework/avocado-vt#553. The patch is used to disable the exception triggered by process.run() not qemu-img. If without the patch avocado-framework/avocado-vt#553, case will be terminated.